pandas/util/mixins.py

```diff
@@ -0,0 +1,28 @@
+"""
+Mixins which can't go to core.common or core.base due to circular imports
+"""
```

Comment:
or if you wan to simply move ``IndexOpsMixin`` to another module. I don't want to create ANOTHER base-like module.